src/transports/encodePolyfill.ts

```diff
@@ -2,13 +2,19 @@ import { CAP_GLOBAL_OBJ } from '../utils/webViewUrl';
 import { utf8ToBytes } from '../vendor';
 
 export const useEncodePolyfill = (): void => {
-  if (!CAP_GLOBAL_OBJ.__SENTRY__) {
-    (CAP_GLOBAL_OBJ.__SENTRY__ as Partial<
+  const globalCarriers = CAP_GLOBAL_OBJ.__SENTRY__;
+
+  if (!globalCarriers) {
+    (globalCarriers as Partial<
       (typeof CAP_GLOBAL_OBJ)['__SENTRY__']
     >) = {};
   }
+  const capacitorSiblingCarrier = globalCarriers?.version && globalCarriers['9.0.0'];
```

Comment:
Q: should we use the version used by the SDK or the newest carrier?

Reply:
used similar implementation as react native, load version using SDK_VERSION from @sentry/core